java/sage/Show.java

```diff
@@ -351,34 +320,35 @@ void setDontLike(boolean x)
   void update(DBObject x)
   {
     Show fromMe = (Show) x;
-    duration = fromMe.duration;
-    lastWatched = fromMe.lastWatched;
-    dontLike = fromMe.dontLike;
-    title = fromMe.title;
-    episodeNameBytes = fromMe.episodeNameBytes;
-    episodeNameStr = fromMe.episodeNameStr;
-    externalID = fromMe.externalID;
-    descBytes = fromMe.descBytes;
-    descStr = fromMe.descStr;
-    categories = fromMe.categories;
-    people = fromMe.people;
-    roles = fromMe.roles;
-    rated = fromMe.rated;
-    ers = fromMe.ers;
-    year = fromMe.year;
-    pr = fromMe.pr;
-    bonuses = fromMe.bonuses;
-    language = fromMe.language;
-    originalAirDate = fromMe.originalAirDate;
-    seasonNum = fromMe.seasonNum;
-    episodeNum = fromMe.episodeNum;
+
```

Comment:
Please undo all of these alignment formatting changes, that's not fitting with the style of all the other code in the SageTV codebase.

Reply:
But do of course remove the lines that use episodeNameBytes of descBytes.